mavros_extras/src/plugins/trajectory.cpp

```diff
@@ -19,6 +19,9 @@
 #include <mavros_msgs/PositionTarget.h>
 #include <nav_msgs/Path.h>
 
+//! Point array length
+static constexpr int LEN = 11;
```

Comment:
Call it `POINTS_LEN`, LEN is too general.

Reply:
And that line may be computed from pymavlink (by cog), but not necessary for now.